astroquery/tests/test_internet.py

```diff
@@ -1,14 +1,24 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from astropy.tests.helper import remote_data
 import requests
+import os
 
 from .. import version
 
+IS_CI = os.environ.get('TRAVIS', False) or os.environ.get('APPVEYOR', False)
+
 
 @remote_data
 def test_net_connection():
-    R = requests.post('http://httpbin.org/post',
-                      headers={'User-Agent':
-                               'astropy:astroquery.{0}'
-                               .format(version.version)})
-    R.raise_for_status()
+    if IS_CI is False:
+        R = requests.post('http://httpbin.org/post',
+                          headers={'User-Agent':
+                                   'astropy:astroquery.{0}'
```

Comment:
looking at this, I think we should say `astroquery-tests` here, too, and only have `astroquery` for actual user usage. @keflavich, what do you think?

Reply:
yes, but it's not terribly important either way here

Reply:
Okay...